composer.json

```diff
@@ -19,7 +19,8 @@
     }
   },
   "require": {
-    "php": ">=5.3.0"
+    "php": ">=5.3.0",
+    "mihaeu/html-formatter": "*"
```

Comment:
I'm not really comfortable with relying on a library a) without specifying a version constraint b) that hasn't been updated in two years. If possible I'd like html-object to stay dependencies-free as that is one of the appeals of it. If I recall you can format HTML in plain PHP:

```php
$html = '<div><h1>Foo</h1></div>';

$dom = new DOMDocument();
$dom->loadHTML($html);
$dom->preserveWhitespace = false;
$dom->formatOutput = true;

$html = $dom->saveXML($dom->getElementsByTagName('body')[0]);
$html = substr($html, 7, -8);
var_dump($html);

//  <div>
//    <h1>Foo</h1>
//  </div>
```